CoastalResilienceNetwork/GeositeFramework
# More unit tests

This adds more unit tests as part of #561.

To test:

1. Uncomment this line https://github.com/CoastalResilienceNetwork/GeositeFramework/blob/layer-selector-v2/src/GeositeFramework/plugins/layer_selector_v2/main.js#L30
2. Run application in browser
3. Verify all tests passed in developer console

![image](https://cloud.githubusercontent.com/assets/43062/13531628/8ca82672-e1f5-11e5-8968-9df93aefbec6.png)
